tools/accuracy_checker/accuracy_checker/metrics/classification.py

```diff
@@ -373,8 +373,8 @@ def evaluate(self, annotations, predictions):
                                           y_pred=all_results,
                                           ).ravel()
 
-        apcer = fp / (tn + fp) if fp != 0 else 0
-        bpcer = fn / (fn + tp) if fn != 0 else 0
+        apcer = fp / (tn + fp)
+        bpcer = fn / (fn + tp)
```

Comment:
it is not totally correct, we can not avoid situation when tn+fp == 0 or fn+tp == 0 (division by zero), please return checks, but change condition

Reply:
I think that situation when both true negatives and false positives equal zero is impossible. However, there could be some problems with data I suppose, so I fixed this.